src/components/inject-main-camera-here.js

```diff
@@ -0,0 +1,8 @@
+AFRAME.registerComponent("inject-main-camera-here", {
```

Comment:
This is the thing I like least about this refactor, but its better than the hoops we were jumping through before. Fixing this probably requires moving the camera rig out of hub.html